src/openms/include/OpenMS/METADATA/ExperimentalDesign.h

```diff
@@ -294,6 +299,15 @@ namespace OpenMS
       // Returns column index of factor
       Size getFactorColIdx( const String &factor) const;
 
+      // Returns the name/ID of the sample. Not necessarily the row index
+      String getSampleName(unsigned sample_row) const;
+
+      // Returns the row index in the sample section for a sample name/ID
+      unsigned getSampleRow(unsigned sample) const;
+
+      /// returns the number of entries in content
```

Comment:
from documentation: not clear what content size refers to or what corresponds to content (e.g., total number of quant channels?)

Reply:
User can look up in the member documentation if there is any